ibis/pandas/execution/constants.py

```diff
@@ -0,0 +1,74 @@
+"""Constants for the pandas backend.
+"""
+
+import operator
+
+import numpy as np
+import pandas as pd
+
+import ibis
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+
+
+JOIN_TYPES = {
+    ops.LeftJoin: 'left',
+    ops.RightJoin: 'right',
+    ops.InnerJoin: 'inner',
+    ops.OuterJoin: 'outer',
+}
+
+
+LEFT_JOIN_SUFFIX = '_ibis_left_{}'.format(ibis.util.guid())
```

Comment:
should these be functions? so the guid is on demand? (IOW what if you do a join of a join)?

Reply:
This isn't an issue because by the time the next join is executed we've removed this suffix. I have some tests for this: `test_multi_join_with_post_expression_filter`.